src/backend/core/api/custom_metrics_exporter.py

```diff
@@ -0,0 +1,198 @@
+        metrics.append(
+            GaugeMetricFamily(
+                prefixed_metric_name("active_users_today"),
+                "Number of active users today",
+                value=user_metrics["active_users_today"],
+            )
+        )
+        metrics.append(
+            GaugeMetricFamily(
+                prefixed_metric_name("active_users_7_days"),
+                "Number of active users in the last 7 days",
+                value=user_metrics["active_users_7_days"],
+            )
+        )
+        metrics.append(
+            GaugeMetricFamily(
+                prefixed_metric_name("active_users_30_days"),
+                "Number of active users in the last 30 days",
+                value=user_metrics["active_users_30_days"],
+            )
+        )
+
+        # -- Document metrics
+        metrics.append(
+            GaugeMetricFamily(
+                prefixed_metric_name("total_documents"),
+                "Total number of documents",
+                value=doc_metrics["total_documents"],
+            )
+        )
+        metrics.append(
+            GaugeMetricFamily(
+                prefixed_metric_name("shared_documents"),
+                "Number of shared documents",
+                value=doc_metrics["shared_docs_count"],
+            )
+        )
+        metrics.append(
+            GaugeMetricFamily(
+                prefixed_metric_name("active_documents_today"),
+                "Number of active documents today",
+                value=doc_metrics["active_docs_today"],
+            )
+        )
+        metrics.append(
+            GaugeMetricFamily(
+                prefixed_metric_name("active_documents_7_days"),
+                "Number of active documents in the last 7 days",
+                value=doc_metrics["active_docs_last_7_days"],
+            )
+        )
+        metrics.append(
+            GaugeMetricFamily(
+                prefixed_metric_name("active_documents_30_days"),
+                "Number of active documents in the last 30 days",
+                value=doc_metrics["active_docs_last_30_days"],
+            )
+        )
+
+        # -- Document oldest/newest timestamps
+        if doc_ages["oldest"]:
+            metrics.append(
+                GaugeMetricFamily(
+                    prefixed_metric_name("oldest_document_date"),
+                    "Timestamp of the oldest document creation date",
+                    value=doc_ages["oldest"].timestamp(),
+                )
+            )
+        if doc_ages["newest"]:
+            metrics.append(
+                GaugeMetricFamily(
+                    prefixed_metric_name("newest_document_date"),
+                    "Timestamp of the newest document creation date",
+                    value=doc_ages["newest"].timestamp(),
+                )
+            )
+
+        # -- User document distribution
+        user_distribution_metric = GaugeMetricFamily(
+            prefixed_metric_name("user_document_distribution"),
```

Comment:
(suggestion) not sure that the  "_distribution" is meaningful?
```suggestion
            prefixed_metric_name("user_documents"),
```

Reply:
Is now `user_documents_total`

Reply:
you shouldn't use a `_total` suffix for gauges, it's normally reserved for counters: https://prometheus.io/docs/specs/om/open_metrics_spec_1_1/#suffixes

Reply:
Oh okay
But it's an accumulated integer count.
So it should not be a gauge then - (code currently exclusively uses gauges).

I'll correct this, thanks